### ISSUE TITLE
Any way to check VIEW privileges similar to table_privs_are?

### ISSUE THREAD

**Issue by jamesmbourne** [theory/pgtap#196]
I am trying to test what privileges a user has on a view in the same way as `has_table_privs`, although from the docs I could not find a function that allows for this.

I tried using `has_table_privs` on the `VIEW` although this did not work:

```
Table example.view_here does not exist
```

Is this functionality that you would consider adding?

**Comment by theory** [comment]
Sure, makes sense. Care to submit a pull request?

**Comment by jamesmbourne** [comment]
@theory Yeah I'll take a look at getting this implemented. I wasn't sure if I was just missing something in my usage of `has_table_privs`
